src/Microsoft.Extensions.Logging.Abstractions/LoggerMessage.cs

```diff
@@ -69,6 +69,26 @@ public static class LoggerMessage
         /// <summary>
         /// Creates a delegate which can be invoked for logging a message.
         /// </summary>
+        /// <param name="logLevel">The <see cref="LogLevel"/></param>
+        /// <param name="eventId">The event id</param>
+        /// <param name="formatString">The named format string</param>
+        /// <returns>A delegate which when invoked creates a log message.</returns>
+        public static Action<ILogger, Exception> Define(LogLevel logLevel, int eventId, string formatString)
```

Comment:
True, but we get LoggingExtensions with two usage patterns inside which doesn't look good.